src/pages/clubs/ClubPage.tsx

```diff
@@ -140,33 +153,31 @@ const ClubPage: React.FC = () => {
           </IonSegmentButton>
         </IonSegment>
 
-        <IonList lines="none">
-          {bookClubData?.discussions.map((discussion, index) => {
-            return (
-              <IonItem key={index}>
-                {selectedSegment === "calendar" ? (
-                  <DiscussionCard
-                    bookClubId={bookClubId}
-                    discussionId={discussion.id}
-                    chapter={discussion.title}
-                    participants={discussion.participants.length}
-                    startTime={discussion.startTime}
-                    endTime={discussion.endTime}
-                    location={discussion.location}
-                    agenda={discussion.agenda}
-                  />
-                ) : (
-                  <ResourceCard
-                    title={"Diva-E's Resource"}
-                    date={"12.12.2022"}
-                    type={"Link"}
-                  />
-                )}
-              </IonItem>
-            );
-          })}
-        </IonList>
-
+        {bookClubData?.discussions.map((discussion, index) => {
```

Comment:
Why did you use <div> instead of <IonList> with <IonItems>? Isn't it better to use ionic components instead of regular HTML tags?